src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_northrend_beasts.cpp

```diff
@@ -131,24 +131,26 @@ enum Events
     EVENT_FIRE_BOMB             = 4,
     EVENT_BATTER                = 5,
     EVENT_HEAD_CRACK            = 6,
+    EVENT_SNOBOLLED             = 7,
```

Comment:
No reason to explicitly specify event ID for events. Just specify `= 1` on the first one, then let the compiler handle the rest.